platform/darwin/src/MGLRasterTileSource.mm

```diff
@@ -33,15 +33,16 @@ - (instancetype)initWithIdentifier:(NSString *)identifier configurationURL:(NSUR
 }
 
 - (instancetype)initWithIdentifier:(NSString *)identifier configurationURL:(NSURL *)configurationURL tileSize:(CGFloat)tileSize {
-    auto source = [self pendingSourceWithIdentifier:identifier configurationURL:configurationURL tileSize:tileSize];
+    NSString *configurationURLString = configurationURL.mgl_URLByStandardizingScheme.absoluteString;
+    auto source = [self pendingSourceWithIdentifier:identifier urlOrTileset:configurationURLString.UTF8String tileSize:tileSize];
```

Comment:
should tileSize be rounded?

Reply:
Good spot. That was there before, I'll re-add.